test/snippet/io/sequence_file/sequence_file_output_fields_trait_1.cpp

```diff
@@ -25,7 +25,7 @@ int main()
 
         // ...
 
-        fout.emplace_back(id, seq_qual);       // note also that the order the argumets is now different, because
+        fout.emplace_back(id, seq_qual);       // note also that the order the arguments is now different, because
```

Comment:
```suggestion
        fout.emplace_back(id, seq_qual, std::get<1>(seq_qual));       // note also that the order of the arguments is now different, because
```

Otherwise, you don't actually write out the quality.

@eseiler can you double-check the grammar 🙈 ?